aequilibrae/project/about.py

```diff
@@ -77,7 +76,7 @@ def add_info_field(self, info_field: str) -> None:
         if has_forbidden:
             raise ValueError(f"{info_field} is not valid as a metadata field. Should be a lower case ascii letter or _")
 
-        with commit_and_close(connect_spatialite(self.__path_to_file)) as conn:
+        with commit_and_close(self.__path_to_file, spatial=True) as conn:
```

Comment:
Does this need `spatial=True`? 

```suggestion
        with commit_and_close(self.__path_to_file) as conn:
```